Dockerfile.devkit

```diff
@@ -54,6 +54,22 @@ RUN wget \
     && 7z e -o/tools "/tmp/${GOCOVER_COBERTURA_FILE}" \
     && chmod +x /tools/gocover-cobertura
 
+ARG GOVC_VERSION=v0.27.4
```

Comment:
If we use the docker version we can have dependabot manage the dep (all of these existing ones are woefully out of date ;( ):

```
FROM docker.io/vmware/govc:v0.27.4 as govc
```

And then below in the copy section:

```
COPY --from=govc /govc /usr/local/bin/
```

Reply:
Thanks for the suggestion. I will make the change.